jerry-core/parser/js/common.c

```diff
@@ -59,7 +59,7 @@ util_print_chars (const uint8_t *char_p, /**< character pointer */
 {
   while (size > 0)
   {
-    printf ("%c", *char_p++);
+    jerry_port_console ("%c", *char_p++);
```

Comment:
I think most jerry_port_console functions should be changed to jerry_port_log with debug level.